client/src/pages/Notifications/NotificationsList/NotificationsList.tsx

```diff
@@ -24,58 +24,49 @@ export function NotificationsList({ notifications, type }: Props): JSX.Element {
   // };
   return (
     <List component="nav" className={classes.root} aria-label="notifications">
-      {notifications.map(
-        (notification: {
-          _id: string;
-          type: string;
-          senderId: {
-            _id: string;
-            username: string;
-          };
-          receiverId: string;
-          contestId: {
-            _id: string;
-            title: string;
-          };
-          submissionId?: string;
-          createdAt: Date;
-          readStatus: boolean;
-          photo: string;
-        }) => (
-          <>
-            <ListItem
-              button
-              key={(notification.type, notification.photo, notification.senderId.username, notification.type)}
-            >
-              <Grid container={true} alignItems="center" justify="center" spacing={2} wrap="nowrap">
-                <Grid item xs={2} container direction="column" justify="flex-end" alignItems="flex-end">
-                  <Avatar alt="Notification Thumbnail" src={notification.photo} />
-                </Grid>
-                <Grid item xs={9}>
-                  <Typography variant="body2" noWrap={type === 'dropdown' ? true : false}>
-                    {notification.type === 'submission' ? (
-                      <span>
-                        <b>{notification.senderId.username}</b> submitted a design to your contest{' '}
-                        <b>{notification.contestId.title}</b>.
-                      </span>
-                    ) : (
-                      <span>
-                        <b>{notification.senderId.username}</b> sent you a message.
-                      </span>
-                    )}
-                    <br />
-                    <i>{timeSince(notification.createdAt)} ago</i>
+      {notifications.map((notification: Notification) => (
+        <>
+          <ListItem
+            button
+            key={(notification.type, notification.photo, notification.senderId.username, notification.type)}
```

Comment:
For the key here, maybe just use the unique notification id? :)

Reply:
I didn't know this. Thanks!